Description: fix spelling and alignment
Origin: Olaf van der Spek <olaf@xwis.net>
Author: Werner Detter <werner@aloah-from-hell.de>
DEP: 3
Bug-Debian: http://bugs.debian.org/712642
Bug-Debian: http://bugs.debian.org/732852
--- a/mytop
+++ b/mytop
@@ -976,7 +976,7 @@
           my $q_diff = ( $STATUS{Questions} - $OLD_STATUS{Questions} );
 #          print("q_diff: $STATUS{Questions} - $OLD_STATUS{Questions}  / $t_delta = $q_diff\n");
 
-          printf(" Sorts: %5.0f qps now: %4.0f Slow qps: %3.1f  Threads: %4.0f (%4.0f/%4.0f) %02.0f/%02.0f/%02.0f/%02.0f \n",
+          printf(" Sorts: %6.0f qps now: %4.0f Slow qps: %3.1f  Threads: %4.0f (%4.0f/%4.0f) %02.0f/%02.0f/%02.0f/%02.0f \n",
 		 ( $STATUS{Sort_rows} - $OLD_STATUS{Sort_rows} ) / $t_delta, 
                  ( $STATUS{Questions} - $OLD_STATUS{Questions} ) / $t_delta,
                  ( # slow now (qps)
@@ -1682,7 +1682,7 @@
   e - explain the query that a thread is running
   E - display current replication error
   f - show full query info for a given thread
-  F - unFilter the dispaly
+  F - unfilter the display
   h - show only a specifc host's connections
   H - toggle the mytop header
   i - toggle the display of idle (sleeping) threads
@@ -1696,7 +1696,7 @@
   r - reset the status counters (via FLUSH STATUS on your server)
   R - change reverse IP lookup
   s - change the delay between screen updates
-  S - change slow quiery hightlighting
+  S - change slow query hightlighting
   t - switch to thread view (default)
   u - show only a specific user
   : - enter a command (not yet implemented)
